routes/feedback.py

```diff
@@ -1,32 +1,36 @@
-
 from config.db import get_database
 from fastapi import APIRouter, Response
-
 from models import events as eventModel
```

Comment:
while technically not a syntax error, the naming is not pythonic. (I know this wasn't done by you but good chance to fix it.)

`eventModel` -> `event_models` is much clearer (it's actually importing _all_ of the `event` models, so plural name makes more sense).